Makefile

```diff
@@ -231,6 +232,15 @@ docker/fv3fit/requirements.txt:
 		external/loaders/setup.py \
 		external/vcm/setup.py
 
+docker/artifacts/requirements.txt:
+	cp constraints.txt $@
+	# this will subset the needed dependencies from constraints.txt
+	# while preserving the versions
+	pip-compile --no-annotate \
+		--output-file docker/artifacts/requirements.txt \
+		external/artifacts/setup.py \
+		projects/microphysics/argo/requirements.in
```

Comment:
hmm. maybe pull this requirements file someplace else. I sort of view the projects/microphysics folder as an event horizon. dependencees go in but not out.